index.js

```diff
@@ -1,5 +1,21 @@
 'use strict';
 
+const AWS = require('aws-sdk');
+const cache = require('./lib/cache');
+
 module.exports = {
+  initialize: (conf) => {
+    let awsConf = {}
+      , configurationKeys = ['accessKeyId', 'secretAccessKey', 'region', 'httpOptions'];
```

Comment:
Does it make sense to bring the proxy lib in here and allow ourselves to signal some way to pass in the raw value and have it converted to whatever aws expects with that lib?